Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -228,11 +228,17 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
 
     @objc public let attribution: Attribution
 
+    @available(*, deprecated, message: "Use purchasesAreCompletedBy instead.")
     @objc public var finishTransactions: Bool {
         get { self.systemInfo.finishTransactions }
         set { self.systemInfo.finishTransactions = newValue }
```

Comment:
Not sure if we can do this for computed variables, but can we move it into Deprecations.swift?

Reply:
Computed vars are A-ok, but `self.systemInfo` is `private`, so we'd have to make it `internal` to move it to the extension. Thoughts?

Reply:
hmmm I'd either go with the new extension or maybe better to just not even bother, it was more of a nitpick but might not be worth the effort